packages/material-ui-styles/test/index.spec.tsx

```diff
@@ -166,7 +166,7 @@ function testGetThemeProps(theme: Theme, props: AppBarProps): void {
       return <div className={className}>Greeted?: {defaulted.startsWith('Hello')}</div>;
     }
   }
-  const StyledMyComponent = styled<typeof MyComponent>(MyComponent)((theme: MyTheme) => ({
+  const StyledMyComponent = styled<typeof MyComponent>(MyComponent)<MyTheme>(({ theme }) => ({
```

Comment:
We try to avoid explicit generic arguments. Does this work with explicit type annotation in the function signature?

Reply:
Yes, it's possible to add type in the function signature. I chose to do it this way because it is lighter, but going to update it.

Reply:
Updated.